Neos.Flow/Classes/Mvc/Controller/ActionController.php

```diff
@@ -260,8 +260,7 @@ public function processRequest(ActionRequest $request): ActionResponse
             $this->initializeView($this->view);
         }
 
-        // We still use a global response here as it might have been changed in any of the steps above
```

Comment:
if someone dared to **replace** `$this->response` above - not just mutate the object!!! - i dont know what i should think.